citi-onboarding/Recorda-one-page
# Feature/feedback section

### **What I did:**
- Added some images to "assets" folder
- Created the "feedback_card" component
- Created the "feedback" page
- Exported the "FeedbackSection" to the react app

### **How to test:**
- git checkout main
- git pull
- git checkout feature/feedback_section
- write "npm run start" on the client directory
- check the feedback section
- review the code
